chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -32,9 +27,10 @@ class ClientsDataSource: NSObject, UITableViewDataSource {
 	}
 
 	func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath) as! ClientTableViewCell
-		cell.client = clients[indexPath.row]
-		cell.updateClient = updateClient
+		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
```

Comment:
The issue is that ClientSettable is just a protocol defining that there must be a *client* property. If I try this:
`if var cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath) as? ClientSettable {
			cell.client = clients[indexPath.row]
			return cell
		}`
I get an error stating that I can't return a ClientSettable, but instead UITableViewCell type is expected. So I've changed it to this:
`		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
		if var cell = cell as? ClientSettable {
			cell.client = clients[indexPath.row]
		}
		return cell
`
But I'm not 100% sure that in this way is ok, as I think the changed variant only works because cell is a reference type property (class, not structure which is value type). I'm not sure it's ok, as if cell was a structure, this code wouldn't work.